spec/cypress/integration/admin-create-administrator.js

```diff
@@ -0,0 +1,39 @@
+describe("Admin user creating another admin user", () => {
+  beforeEach(() => {
+    cy.login("admin");
+  });
+
+  it("should show a create administrator button", () => {
+    cy.visit("/admin/administrators");
+    cy.get("a.govuk-button").contains("Create a new administrator");
+  });
+
+  it("should create a new user", () => {
+    const fullName = "John Smith";
+    const email = "j.smith@example.com";
+
+    cy.visit("/admin/administrators");
+    cy.get("a.govuk-button").contains("Create a new administrator").click();
+
+    cy.get("h1").should("contain", "Create a new administrator user");
```

Comment:
I haven't actually ran these test so can't test this myself, but it's usually better to test the url with `cy.location('pathname')` if it has changed - if it hasn't, that's when to test the heading element. Headings change more than URLs :)